Shared/ViewModels/VideoPlayerViewModel.swift

```diff
@@ -68,7 +68,9 @@ class VideoPlayerViewModel: ViewModel {
         let configuration = VLCVideoPlayer.Configuration(url: playbackURL)
         configuration.autoPlay = true
         configuration.startTime = .seconds(max(0, item.startTimeSeconds - Defaults[.VideoPlayer.resumeOffset]))
-        configuration.audioIndex = .absolute(selectedAudioStreamIndex)
+        if self.audioStreams[0].path != nil {
```

Comment:
This is simply a hack to get the correct audio track to be played by default, correct? In reality, there needs to be more done to sort this out, I think: https://github.com/jellyfin/Swiftfin/issues/926

Reply:
Thanks. We had conversed in private to just merge this as-is to get this work in and then fixes/additional work can be done afterwards.